Library/Homebrew/test/cmd/home_spec.rb

```diff
@@ -20,6 +24,21 @@
     Cask::CaskLoader.load(local_caffeine_path).homepage
   }
 
+  let(:local_transmission_path) {
+    cask_path("local-transmission")
+  }
+
+  let(:local_transmission_homepage) {
+    Cask::CaskLoader.load(local_transmission_path).homepage
+  }
+
+  it "opens the project page when no formula or cask is specified" do
```

Comment:
Because these are integration tests (and therefore really slow) we want to have as few as these as possible to cover all necessary functionality.